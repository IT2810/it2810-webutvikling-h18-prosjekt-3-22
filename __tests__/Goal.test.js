import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer'
import Goal from '../src/components/goal.js';

//creates snapshots from goal-component
test('Take snapshot of Goal', () => {
  const snap = renderer.create( <
    Goal / >
  ).toJSON();
  expect(snap).toMatchSnapshot();
});