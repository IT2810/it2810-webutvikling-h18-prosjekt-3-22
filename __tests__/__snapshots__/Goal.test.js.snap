// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Goal-page test Take snapshot of goal-component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 42,
        "padding": 10,
      }
    }
  >
    Goal
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "marginBottom": 10,
      }
    }
  >
    Steps taken today: 
    0
    /
    10000
  </Text>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      Walk 
      10000
       steps to reach your goal.
    </Text>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "marginTop": 30,
      }
    }
  >
    Set a new daily goal
  </Text>
  <TextInput
    allowFontScaling={true}
    clearButtonMode="always"
    onChangeText={[Function]}
    onSubmitEditing={[Function]}
    placeholder="10 000 steps"
    style={
      Object {
        "borderColor": "gray",
        "borderWidth": 1,
        "height": 50,
        "marginTop": 10,
        "padding": 15,
        "textAlign": "center",
        "width": "80%",
      }
    }
    testID="goalInput"
  />
</View>
`;
